### DISCUSSION TITLE
V9 support for Vue 2? No v-tc or <i18n path="" tc> in v8

### DISCUSSION THREAD

**Discussion by darkylmnx** [intlify/vue-i18n#394] [Q&A] [answered]
Hi, I was looking on v8 for the $tc equivalent in directives, something like `v-tc=" ... "`, I didn't fint it. And looking in the issues of v8, @kazupon you said v9 had component interpollation for pluralization here https://github.com/kazupon/vue-i18n/issues/195#issuecomment-791116415

So I was wondering, does v9 support Vue 2?
I checked in v9 docs and didn't find anything about that.

**Comment by kazupon** [comment, marked as answer]
sorry, poor documentation
In v9,  component interpolation (`Translation Component (i18n-t))` support pluralization.
you can use `plural` prop.

example test code
https://github.com/intlify/vue-i18n-next/blob/8076303ba178622dfc2de41075a061cc6eb231dc/packages/vue-i18n/test/components/Translation.test.ts#L111-L134

**Reply by darkylmnx** [reply]
no problem for the documentation. But does v9 work with Vue 2?

**Reply by kazupon** [reply]
Unfortunately, v9 doesn’t support for Vue 2. 
However, v9 API almost support v8.x API I/F

See the docs:
https://vue-i18n.intlify.dev/guide/migration/breaking.html

**Comment by zhouxianjun** [comment]
@darkylmnx are you resolve？
